docs/examples/Space_Robot.rst

```diff
@@ -0,0 +1,538 @@
+
+.. jupyter-execute::
+
+    Fx_mag, Fy_mag, Fz_mag = me.dynamicsymbols('Fmag_x, Fmag_y, Fmag_z')
+    
+    Fx = Fx_mag * ISS.x
+    Fy = Fy_mag * ISS.y
+    Fz = Fz_mag * ISS.z
+    
+    Fx, Fy, Fz
+
+
+
+
+
+Torques
+~~~~~~~
+
+.. jupyter-execute::
+
+    T1_mag, T2_mag, T3_mag = me.dynamicsymbols('Tmag_1, Tmag_2, Tmag_3')
+    
+    T1 = T1_mag * B.x
+    T2 = T2_mag * B.y
+    T3 = T3_mag * B.z
+    
+    T1, T2, T3
+
+
+
+
+
+Kane’s Method
+-------------
+
+.. jupyter-execute::
+
+    kdes = [z1.rhs - z1.lhs, z2.rhs - z2.lhs, z3.rhs - z3.lhs, z4.rhs - z4.lhs, z5.rhs - z5.lhs, z6.rhs - z6.lhs]
+
+.. jupyter-execute::
+
+    body = me.RigidBody('body', C, B, m, (I, C))
+    bodies = [body]
+
+.. jupyter-execute::
+
+    loads = [
+             (C, Fx),
+             (C, Fy),
+             (C, Fz),
+             (B, T1),
+             (B, T2),
+             (B, T3)
+            ]
+
+.. jupyter-execute::
+
+    kane = me.KanesMethod(ISS, (x, y, z, q1, q2, q3), (ux, uy, uz, u1, u2, u3), kd_eqs=kdes)
+
+.. jupyter-execute::
+
+    fr, frstar = kane.kanes_equations(bodies, loads=loads)
+
+
+
+Simulation
+----------
+
+.. jupyter-execute::
+
+    sys = System(kane)
+
+.. jupyter-execute::
+
+    sys.constants_symbols
+
+
+
+.. jupyter-execute::
+
+    sys.constants = {
+                     Ix: 0.1083,
+                     Iy: 0.1083,
+                     Iz: 0.1083,
+                     m: 7
+                    }
+
+.. jupyter-execute::
+
+    sys.constants
+
+
+
+.. jupyter-execute::
+
+    sys.times = np.linspace(0.0, 50.0, num=1000)
+
+.. jupyter-execute::
+
+    sys.coordinates
+
+
+
+.. jupyter-execute::
+
+    sys.speeds
+
+
+.. jupyter-execute::
+
+    sys.states
+
+
+
+
+.. jupyter-execute::
+
+    sys.initial_conditions = {
+                              x: 0.0,
+                              y: 0.0,
+                              z: 0.0,
+                              q1: 0.0,
+                              q2: 0.0,
+                              q3: 0.0,
+                              ux: 0.2,
+                              uy: 0.0,
+                              uz: 0.0,
+                              u1: 0.0,
+                              u2: 0.0,
+                              u3: 0.5
+                             }
+
+.. jupyter-execute::
+
+    sys.specifieds_symbols
+
+
+
+
+.. jupyter-execute::
+
+    sys.specifieds = {
+                      Fx_mag: 0.0,
+                      Fy_mag: 0.0,
+                      Fz_mag: 0.0,
+                      T1_mag: 0.0,
+                      T2_mag: 0.0,
+                      T3_mag: 0.0
+                     }
+
+.. jupyter-execute::
+
+    states = sys.integrate()
+
+.. jupyter-execute::
+
+    import matplotlib as mpl
+    mpl.rcParams['figure.dpi'] = 200
+    mpl.rc('font',**{'family':'serif','sans-serif':['Computer Modern Roman']})
+    ## for Palatino and other serif fonts use:
+    #rc('font',**{'family':'serif','serif':['Palatino']})
+    mpl.rc('text', usetex=True)
```

Comment:
I think this is causing issues. We don't have latex installed on the CI system, so removing all these mpl.rc settings is probably preferable.